extensions/amp-story/0.1/system-layer.js

```diff
@@ -222,6 +222,12 @@ export class SystemLayer {
         this.onUnmuteAudioClick_(e);
       }
     });
+
+    this.win_.addEventListener('keyup', e => {
+      if (!this.closeBookendBtn_.hasAttribute('hidden') && e.key == 'Escape') {
```

Comment:
I'm sure we have a constant for escape; see `../../../src/utils/key-codes`

Reply:
Done